README.md

```diff
@@ -40,13 +40,9 @@ e.g.
 ### Setup in SendGrid portal
 
 1. Log in at https://app.sendgrid.com. Username is "hmcts-\<name-specified-in-configuration\>". Retrieve password from vault.
-2. Click 'Authenticate your domain'
-2. Select 'Other Host (Not listed)
-3. Type 'Azure'
-4. brand links: normally we choose no (the default)
-5. click 'Next'
-6. enter the from domain, e.g. mail-sscs-nonprod.platform.hmcts.net
-7. click 'Next'
+2. Browse to Settings --> Sender Authentication on the left side of the menu
+3. Click on the domain in question
```

Comment:
⛔ This section should be removed as they don't need to setup in the portal anymore but the docs need updating in this file for the new method.

Reply:
Do we need to specify instruction how to setup the new account? so for examle,
1. Make sure the new account you are setting up has correct domain  name and zone name, check other accounts as examples (e.g. access-management.tf)
2. Once changes has been mereged and the pipeline applies them successfully, please follow below steps.
   a. Log in at https://app.sendgrid.com. Username is "hmcts-\<name-specified-in-configuration\>". Retrieve password from vault.
   b. Browse to Settings --> Sender Authentication on the left side of the menu
   c. Then tick 'I've added these records.' and click 'Verify'.

If successful then DMARC is all setup.

Reply:
>  Then tick 'I've added these records.' and click 'Verify'.

Is there not a way to verify in the pipeline? if not then yes we do need that, but maybe we can add an API call at the end of apply that just triggers verify on all accounts?